client/src/components/molecules/RequestsTable.tsx

```diff
@@ -56,8 +66,20 @@ const RequestsTable: FunctionComponent<Props> = (props: Props) => {
       }, []);
       
      //console.log(sortedRequests);
```

Comment:
lol could u also help remove this (even tho it's not urs)

Reply:
Removed.